src/ui/MLFacetManager/MLFacetManager.ts

```diff
@@ -0,0 +1,183 @@
+import { Component } from '../Base/Component';
+import { MLFacet } from '../MLFacet/MLFacet';
+import { InitializationEvents } from '../../events/InitializationEvents';
+import { QueryEvents, IQuerySuccessEventArgs } from '../../events/QueryEvents';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { exportGlobally } from '../../GlobalExports';
+import { find } from 'underscore';
+import { IFacetResponse } from '../../rest/Facet/FacetResponse';
+import { $$ } from '../../utils/Dom';
+import { Utils } from '../../utils/Utils';
+import { ComponentOptions } from '../Base/ComponentOptions';
+
+export interface IMLFacetManagerOptions {
+  enableReorder?: boolean;
+  onUpdate?: IMLFacetManagerOnUpdate;
+  compareFacets?: IMLFacetManagerCompareFacet;
+}
+
+export interface IMLFacetManagerOnUpdate {
+  (facet: MLFacet, index: number): void;
+}
+
+export interface IMLFacetManagerCompareFacet {
+  (facetA: MLFacet, facetB: MLFacet): number;
+}
+
+/**
+ * The `MLFacetManager` component is meant to be a parent for multiple [MLFacet]{@link MLFacet} components.
+ * It allows more control over the rendering and ordering of the children [MLFacet]{@link MLFacet} components.
+ */
+export class MLFacetManager extends Component {
+  static ID = 'MLFacetManager';
+  static doExport = () => exportGlobally({ MLFacetManager });
+
+  /**
+   * The options for the MLFacet
+   * @componentOptions
+   */
+  static options: IMLFacetManagerOptions = {
+    /**
+     * Whether to allow the reordering of facets based on Coveo ML and index ranking scores.
+     *
+     * **Default:** `true`
+     */
+    enableReorder: ComponentOptions.buildBooleanOption({ defaultValue: true, section: 'Filtering' }),
```

Comment:
Yes that was the idea, let's say you just want to hide or collapse the facets instead of reordering them, you could do it here, but there might be other features in the future that will also be useful.